io/js/src/main/scala/fs2/io/file/Path.scala

```diff
@@ -98,9 +98,21 @@ final case class Path private[file] (override val toString: String) extends Path
 object Path extends PathCompanionApi {
   private[file] val sep = facade.path.sep
 
+  /*
+   * NOTE: It seems like scala js does not rewrite this to a loop?
+   * Call stack limit is reached if there are too many separators.
+   */
+  @tailrec
+  private[file] def dropTrailingSep(path: String): String =
+    // Drop separator only if there is something else left
+    if (path.endsWith(sep) && path.length > sep.length)
```

Comment:
I don't like to Officially:tm: support Windows 🤮 but its path can look like `C:\` which I think would get re-written to `C:`. I have no idea if that's good or bad 😅

Reply:
Doing some tests on **Powershell**

```
PS C:\Users\PC> cd c:
PS C:\Users\PC> cd C:
PS C:\Users\PC> cd C:\
PS C:\>
```

Reply:
Used node API exclusively